src/app/loading/components/loading-container/loading-container.component.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { LoadingContainerComponent } from './loading-container.component';
+
+describe('loading container (isolated)', () => {
+  let loadingContainer: LoadingContainerComponent;
```

Comment:
I think a better way to test this component is to see if the various elements are visible or invisible depending on the various loading conditions.